deneme.js

```diff
@@ -9,3 +9,4 @@ class Person {
         console.log(`Hello ${person.name}, this is ${this.name}`)
     }
 }
+askljdksajda
```

Comment:
thank you for adding a new line here it helps to identify initialization phase of the variables

Reply:
this is a single comment